riscv/triggers.cc

```diff
@@ -133,6 +134,9 @@ match_result_t mcontrol_t::memory_access_match(processor_t * const proc, operati
   }
 
   if (simple_match(xlen, value)) {
+    /* This is OK because this function is only called if the trigger was not
+     * inhibited by the previous trigger in the chain. */
+    hit = true;
```

Comment:
IIUC this is implementing the "temperature coding" discussed [here](https://github.com/riscv/riscv-debug-spec/issues/550) and [here](https://github.com/riscv/riscv-debug-spec/pull/593), correct? It would be good to explicitly mention that choice in the commit log or comments.

Reply:
That's right. I've added a comment.